department-of-veterans-affairs/caseflow
# Queue | Filter decided issues from SelectDispositionsView

Resolves #5466 

### Description
This removes issues that have already been decided from the attorney checkout flow.

### Acceptance Criteria 
- [ ] Issues with [dispositions](https://github.com/department-of-veterans-affairs/caseflow/blob/master/constants/VACOLS_DISPOSITIONS_BY_ID.json) A through X are filtered from the Select Dispositions View

### Testing Plan
1. Go to `/queue` as `BVAAABSHIRE` (a judge, but they have an appeal with decided issues)
2. Go to detail view for Grover E. Kerluke (`/queue/appeals/3449656`), confirm the last 3 issues have disposition `A - Advance Allowed in Field`
3. Start Draft Decision checkout flow, confirm those issues are missing from Select Dispositions view

![filter_decided_issues](https://user-images.githubusercontent.com/8533004/39878258-e01037fe-5445-11e8-94c3-6a0b264c7dd2.gif)
